dashboard/src/components/Table/Chevron.tsx

```diff
@@ -0,0 +1,7 @@
+import { MdChevronRight } from 'react-icons/md';
+
+export const ChevronRightAnimate = (): JSX.Element => {
```

Comment:
NIT: Since this is being used by `Accordion` it shouldn't be in the `Table` folder
Maybe `@/components/AnimatedIcons`